### PR TITLE
Initial Import

### PR DESCRIPTION
**Pull request by pruzanov**

### CODE REVIEW

**Comment by pruzanov** [comment]
retest this please

File: wgsmetrics.wdl [hunk 1]
```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+workflow wgsMetrics {
+input {
+    File    inputBam
+    String? refFasta = "$HG19_ROOT/hg19_random.fa"
+    String? outputFileNamePrefix = ""
+    String? stringencyFilter = "LENIENT"
+    String? modules = "java/8 picard/2.19.2 hg19/p13"
```
Comment (mlaszloffy) [inline wgsmetrics.wdl]: Move to task parameters

File: wgsmetrics.wdl [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# ==========================================
+#  TASK 1 of 1: collect WGS metric
+# ==========================================
+task collectWGSmetrics {
+input { 
+        File   inputBam
+        String? refFasta
+        String? metricTag  = "WGS"
+        String? filter     = "LENIENT"
+        String? outputPrefix = ""
```
Comment (mlaszloffy) [inline wgsmetrics.wdl]: Provide default `outputPrefix` otherwise the output file would be named `.WGS`

**Comment by pruzanov** [comment]
retest this please

**Comment by pruzanov** [comment]
retest this please

File: wgsmetrics.wdl (hunk 1 above)
Comment (pruzanov) [reply wgsmetrics.wdl]: this is done

File: wgsmetrics.wdl (hunk 2 above)
Comment (pruzanov) [reply wgsmetrics.wdl]: fixed, see line 9 of wgsmetrics.wdl

**Review by lheisler — APPROVED** [review]
I've not looked at enough of these to catch issues or even have a clear sense yet of best style.
But I walked through the logic and I can't seen anything of concern.  Will approve, assuming others with a better eye are looking at it too.

**Review by mlaszloffy — APPROVED** [review]

**Comment by mlaszloffy** [comment]
retest this please

**Pull request merged**